### PR TITLE
Add custom script tutorial 

### PR DESCRIPTION
**Pull request by cnsidero**

### CODE REVIEW

File: docs/source/user/tutorials/cpl_script.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+docs (:mod:`caelus`) for a complete listing of modules and associated
+functionality.
+
+.. code-block:: python
+
+    from caelus.config.cmlenv import cml_get_version
+    from caelus.run.cmd import CaelusCmd
+    from caelus.run.core import get_mpi_size
+    from caelus.post.logs import LogProcessor, SolverLog
+    from caelus.post.plots import CaelusPlot
```
Comment (sayerhs) [inline docs/source/user/tutorials/cpl_script.rst]: `DictFile` used below seems to be missing from imports?

File: docs/source/user/tutorials/cpl_script.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+An automated way to detect and set up a parallel run is to check for a
+:file:`system/decomposeParDict` file, use the
+:class:`~caelus.io.caelusdict.CaelusDict` class to retrieve the 
+`numberOfSubdomains` parameter, and set the number of MPI ranks
+to run applications with.
+
+.. code-block:: python
+
+    if os.path.isfile("system/decomposeParDict"):
```
Comment (sayerhs) [inline docs/source/user/tutorials/cpl_script.rst]: This section can be written as 

```
decomp_dict = DecomposeParDict.read_if_present()

parallel = True if decomp_dict['numberOfSubDomains'] > 1 else False
```

Comment (sayerhs) [reply docs/source/user/tutorials/cpl_script.rst]: In general, minimize use of `load`

Comment (cnsidero) [reply docs/source/user/tutorials/cpl_script.rst]: Will do.

**Comment by cnsidero** [comment]
Fix 11241c75198d51f10f9ac9242851eb2387915b67 commited.

**Pull request merged**